packages/expo/expo-module.config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "platforms": ["android", "ios"],
```

Comment:
not super familiar with native modules, but would this in any way prevent the Expo package from being used with Expo Web?